expert/ezproxy.cfg

```diff
@@ -841,4 +852,3 @@ Host https://doi.org
 Host dx.doi.org
 Host https://dx.doi.org
 Domain doi.org
-
```

Comment:
Because of the way this configuration is concatenated by oclc for the live service, I'd feel better if we had another line folllowing our config. Doesn't have to be blank if your editor is nagging you, maybe a comment like:
```
# **************** DO NOT DELETE * INTENTIONAL TRAILING LINE * DO NOT DELETE *****************
```

Reply:
I think this happened because I edited the file with vim instead of my usual editor. I'll add an `.editorconfig` file to prevent this from happening in the future